modules/apps/27-interchain-accounts/genesis.go

```diff
@@ -22,7 +22,7 @@ func InitGenesis(ctx sdk.Context, keeper keeper.Keeper, state types.GenesisState
 
 // ExportGenesis exports transfer module's portID into its geneis state
 func ExportGenesis(ctx sdk.Context, keeper keeper.Keeper) *types.GenesisState {
-	portID := keeper.GetPort(ctx)
+	portID := keeper.GetPort(ctx, types.PortID)
```

Comment:
Export Genesis should now be doing a range query on the prefix key and exporting all portIDs. Maybe we can add a todo and link the genesis ica issue?

Reply:
Yep np! Done